src/components/Modal/index.tsx

```diff
@@ -27,14 +28,19 @@ const Modal = ({
   shouldCloseOnOverlayClick = false,
   modalHeading,
   openModal,
-  closeModal
+  closeModal,
+  fireAfterOpening
 }: ModalProps) => {
   const handleOpenModal = () => {
     if (!scroll) noScroll.on();
 
     if (openModal) {
       openModal();
     }
+
+    if (fireAfterOpening) {
+      fireAfterOpening();
+    }
```

Comment:
I'm a little unsure of the `fireAfterOpening` is created, would just calling the function needed on close work?  Normally we run the resets when modals close, rather than after opening.  Maybe we can sync to talk through ti